### PR TITLE
Fix start.hugegraph.sh timeout when config authentication

### PR DESCRIPTION
**Pull request by Linary**
Change-Id: Ic22531cead13f3698791070a044d66eacb1c26ee

### CODE REVIEW

File: hugegraph-dist/src/assembly/static/bin/start-hugegraph.sh [hunk 1]
```diff
@@ -54,6 +54,38 @@ REST_SERVER_URL=`read_property "$CONF/rest-server.properties" "restserver.url"`
...
+        if [[ -z "${USERNAME}" || -z "${PASSWORD}" ]]; then
+            echo "Invalid config for auth.user_tokens"
+            exit 1
+        fi
+        # trim space
+        USERNAME=`echo ${USERNAME} | xargs`
+        PASSWORD=`echo ${PASSWORD} | xargs`
+    fi
+fi
+
```
Comment (zhoney) [inline hugegraph-dist/src/assembly/static/bin/start-hugegraph.sh]: prefer to move it into util.sh as a function

File: hugegraph-dist/src/assembly/static/bin/start-hugegraph.sh [hunk 2]
```diff
@@ -54,6 +54,38 @@ REST_SERVER_URL=`read_property "$CONF/rest-server.properties" "restserver.url"`
 check_port "$GREMLIN_SERVER_URL"
 check_port "$REST_SERVER_URL"
 
+
```
Comment (zhoney) [inline hugegraph-dist/src/assembly/static/bin/start-hugegraph.sh]: delete one empty line

**Comment by javeme** [comment]
If username and password is stored in db, we can't get it from shell. Prefer to add a specific user or a specific api.

File: hugegraph-dist/src/assembly/static/conf/hugegraph.properties [hunk 3]
```diff
@@ -21,6 +21,9 @@ serializer=binary
 
 store=hugegraph
 
+search.text_analyzer=jieba
+search.text_analyzer_mode=INDEX
```
Comment (javeme) [inline hugegraph-dist/src/assembly/static/conf/hugegraph.properties]: also update example and test

**Pull request merged**